packages/template-retail-react-app/app/components/drawer-menu/index.jsx

```diff
@@ -79,17 +83,18 @@ const STORE_LOCATOR_HREF = '/store-locator'
  */
 const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
     const intl = useIntl()
-    const customer = useCustomer()
+    const customerType = useCustomerType()
```

Comment:
One thing that i noticed for the `useCustomerType` helper hook, is that you always do

`customerType === 'guest' && ...` or `customerType === 'register' && ...`

This is a bit tedious, and I'm going to propose that we change this to `useIsGuest` + `useIsRegistered`, in another PR..

Reply:
done